master/internal/task/allocation.go

```diff
@@ -172,7 +172,7 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 		} else {
 			ctx.Respond(errors.New(fmt.Sprintf("unknown container %s", msg.ContainerID)))
 		}
-	case sproto.ReleaseResources:
+	case sproto.ReleaseResources, sproto.ChangeRP:
```

Comment:
nit: semantically, the allocation isn't really changing RPs. I think it trial.go should just send it a `task.Terminate` signal.